main/src/main/java/cgeo/geocaching/connector/internal/InternalConnector.java

```diff
@@ -45,10 +46,11 @@ public class InternalConnector extends AbstractConnector implements ISearchByGeo
 
     // prefix - must not contain regexp characters
     public static final String PREFIX = "ZZ";
+    private static final String GEOCODE_CHARS = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ";
```

Comment:
Suggest to leave out I, O and S (to avoid confusion with 1, 0 and 5)

Reply:
This one?

Reply:
staying with {4} I don't want to reduce the available range. And also those codes are usually not read/written, so don't see the risk of confusing those characters. (for UDCs the geocode could probably be completly hidden from the user)

Reply:
it's a reduction of possible values from 1,7 to 1,2 million (base 31 would be 0,9 million), statistically this means that for ~1300 geocodes generated the likelyhood of 1 conflicting is 50%. Of course this number is not relevant locally (becase there we can check for dups but only on exchanging GPX files
(36^5 would be ~9000, 31^5 6300)